validator/client/grpc-api/grpc_validator_client.go

```diff
@@ -33,7 +33,7 @@ func (c *grpcValidatorClient) Duties(ctx context.Context, in *ethpb.DutiesReques
 		dutiesResponse, err := c.beaconNodeValidatorClient.GetDutiesV2(ctx, in)
 		if err != nil {
 			if status.Code(err) == codes.Unimplemented {
-				log.Warn("beaconNodeValidatorClient.GetDutiesV2() returned status code unavailable, falling back to GetDuties")
+				log.Warn("BeaconNodeValidatorClient.GetDutiesV2() returned status code unavailable, falling back to GetDuties")
```

Comment:
```suggestion
log.Warn("GetDutiesV2 returned status code unavailable, falling back to GetDuties")
```
So that the type name doesn't get capitalized (and the type logged was incorrect anyway)